tests/FilesystemTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Artifacts\Tests;
+
+use Keboola\Artifacts\Filesystem as ArtifactsFilesystem;
+use Keboola\Temp\Temp;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+
+class FilesystemTest extends TestCase
+{
+    public function testFilesystemInitsDirectoriesStructure(): void
+    {
+        $temp = new Temp();
+        $artifactsFilesystem = new ArtifactsFilesystem($temp);
+
+        $path = $artifactsFilesystem->getTmpDir();
+        self::assertDirectoryExists($path);
+        self::assertSame(
+            sprintf('%s/tmp', $temp->getTmpFolder()),
+            $path
+        );
+
+        $path = $artifactsFilesystem->getDataDir();
+        self::assertDirectoryExists($path);
+        self::assertSame(
+            sprintf('%s/data', $temp->getTmpFolder()),
+            $path
+        );
+
+        $path = $artifactsFilesystem->getArtifactsDir();
+        self::assertDirectoryExists($path);
+        self::assertSame(
+            sprintf('%s/data/artifacts', $temp->getTmpFolder()),
+            $path
+        );
+
+        $path = $artifactsFilesystem->getCurrentDir();
+        self::assertDirectoryExists($path);
+        self::assertSame(
+            sprintf('%s/data/artifacts/current', $temp->getTmpFolder()),
+            $path
+        );
+
+        $path = $artifactsFilesystem->getRunsDir();
+        self::assertDirectoryDoesNotExist($path);
```

Comment:
Je spravne ze se ta `runs` slozka nema zakladat? Ja bych rekl spis ze je to chyba.
Na funkci ty libky to nema vliv.